src/components/AnswerCard.js

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { FiArrowUpRight } from 'react-icons/fi'
+import styled from "styled-components";
+import graphql from 'babel-plugin-relay/macro'
+import { createFragmentContainer } from 'react-relay'
+
+import { Text } from '../styles'
+
+function AnswerCard(props) {
+  const { id,
+    answer,
+    text,
+    citationUrl,
+    citationTitle,
+    credibleCount,
+    notCredibleCount } = props.answer
```

Comment:
Wrapping object keys like this is a little hard to read because alignment doesn't match. What about:

```javascript
const {
  id,
  answer
} = props.answer
```

I also prefer writing explicit props when possible, like:

```javascript
function answerCard ({ answer }) {
  const { ... } = answer
}
```